api/src/utils/get-accountability-for-token.ts

```diff
@@ -49,8 +49,8 @@ export async function getAccountabilityForToken(
 
 			accountability.user = user.id;
 			accountability.role = user.role;
-			accountability.admin = user.admin_access === true || user.admin_access == 1;
-			accountability.app = user.app_access === true || user.app_access == 1;
+			accountability.admin = user.admin_access === true || user.admin_access === 1;
+			accountability.app = user.app_access === true || user.app_access === 1;
```

Comment:
(same as the comment for line 26-27)

Reply:
Yeah, this is a great example already, what can be caught with this rule! The type itself only covers `boolean | number` but somehow we're expecting it to be a string as well...
So we either need to update the type or remove test case for strings (I'm really curious why we expected it to also be a string in some cases).